src/seeds.history.cpp

```diff
@@ -80,42 +89,27 @@ void history::addcitizen(name account) {
   size_change("citizens.sz"_n, 1);
 }
 
-void history::addreputable(name organization) {
+void history::updatestatus (name account, name scope) {
   require_auth(get_self());
 
-  auto uitr = users.find(organization.value);
-  check(uitr != users.end(), "no user found");
-  check(uitr -> type == name("organisation"), "the user type must be organization");
-
-  reputables.emplace(_self, [&](auto & org){
-    org.id = reputables.available_primary_key();
-    org.organization = organization;
-    org.timestamp = eosio::current_time_point().sec_since_epoch();
+  account_status_tables acctstatus(get_self(), scope.value);
+  
+  acctstatus.emplace(_self, [&](auto & item){
+    item.id = acctstatus.available_primary_key();
+    item.account = account;
+    item.timestamp = eosio::current_time_point().sec_since_epoch();
```

Comment:
@IanMendozaJaimes don't you have to remove it from the old scope when you add it to the new scope?

Reply:
Oh this just records the time when an org became (status) - cool! No questions!